analysis/staticAnalysisReport.Rmd

```diff
@@ -0,0 +1,171 @@
+---
+title: "Eval Static Analysis"
+author: "Chakshu Goyal"
+date: "07/06/2018"
+output:
+  html_document: default
+  pdf_document: default
+---
+
+## 
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(ggplot2)
+library(tidyverse)
+library(tools)
+#knitr::kable(head(arrange(evalCntPerPkg, desc(freq)) , 20) , col.names = c("Package" , "Number of evals"))
```

Comment:
If it is not important that do not keep it here

Reply:
Acknowledged!